tests/constants/data.constant.ts

```diff
@@ -0,0 +1,84 @@
+import { UserRole } from "../../src/app/core/models/user.model"
+
+export const csvString = `
+role,name,team
+student,JunWei96,CS2103T-W12-3
+admin,damithc,
+tutor,anubh-v,CS2103T-W12-3
+admin,geshuming,
+tutor,testathorTutor,CS2103T-W12-3
+tutor,testathorTutor,CS2103T-W12-4
+tutor,anotherTutor,CS2103T-W12-4
+tutor,testathorTutor,CS2103T-W12-3
+student,testathorStudent,CS2103T-W12-3
+student,damithc,CS2103T-W12-3
+student,somestudent,CS2103T-W12-4
+student,RonakLakhotia,CS2103T-W12-4
+student,ptvrajsk,CS2103T-W12-3
```

Comment:
Would it be better if we used more generic data as test data? It would make it simpler and easier to understand and modify them. [Example in my PR](https://github.com/CATcher-org/CATcher/blob/ec9dcb91fbf57f064e68e282a5269f71f9985e06/tests/constants/data.constants.ts)

Reply:
Thanks for the review. That's a good point. I will change this

Reply:
On further thought, perhaps we should stick with realistic names to help catch problems that occur in real usage.

I think we should also remove names like `testathorStudent`

How about we standardise the data in PR #371 first, before continuing this PR?